.gitignore

```diff
@@ -26,6 +26,7 @@ yarn-error.log*
 TODO.md
 packages/testing/
 packages/puppeteer-extra2/
+packages/puppeteer-extra-old/
```

Comment:
Was this added by accident?

Reply:
Nah, I'm working on the playwright rewrite and that folder is excluded in the new branch already :D

Reply:
I could have selectively committed things instead but was lazy :)